service/handlers.go

```diff
@@ -15,6 +15,15 @@ import (
 
 var pubkey *rsa.PublicKey
 
+// JWT Auth related constants
+const (
+	ContextToken = "token"
+	ContextClaims = "claims"
+	ClaimUsername = "username"
```

Comment:
Can you one empty line between claims and contexts? Makes it easier to understand.